noir-projects/aztec-nr/aztec/src/test/helpers/authwit.nr

```diff
@@ -10,11 +10,11 @@ use protocol_types::{
     constants::CANONICAL_AUTH_REGISTRY_ADDRESS, traits::ToField,
 };
 
-pub unconstrained fn add_private_authwit_from_call<T, let M: u32>(
+pub unconstrained fn add_private_authwit_from_call<let M: u32, let N: u32, T>(
```

Comment:
Aligned here the ordering of generics with PrivateCall. While not really necessary I was not sure if M and N is mixed up somewhere when I was debugging what turned out to be an unrelated issue.